src/apps/samples/views.py

```diff
@@ -20,49 +26,43 @@ def get(self):
 
 
 @api.route('/new')
-class New(Resource):
+class New(BaseView):
     def post(self):
-        title = request.form.get('title')
-        body = request.form.get('body')
-
-        if not title or not body:
-            abort(400, 'bad request')
-
-        Sample.create(title=title, body=body)
-        return 200
+        request_data = request.get_json()
+        data, errors = sample_schema.load(request_data)
+        if errors:
+            raise UnprocessableEntity(errors)
+        Sample.create(title=data.get('title'), body=data.get('body'))
+        return 201
 
 
 @api.route('/get/<string:sample_id>')
-class Get(Resource):
+class Get(BaseView):
     def get(self, sample_id):
         sample = Sample.find_by_id(int(sample_id))
         if not sample:
             abort(404, 'not found')
-
-        return {'sample': {
-            'title': sample.title,
-            'body': sample.body,
-        }}
+        return self.render_json_response(sample, status_code=200, schema=sample_schema)
```

Comment:
```suggestion
        return sample_schema.jsonify(sample)
```
như này có được ko ông ơi? (built in)

Reply:
cái hàm tôi tự viết render_json_response , với lại cái dump schema nó còn có dump many=True với False các kiểu nữa

Reply:
với json response thì có thể dùng hàm built-in `flask.jsonify` mà ông
còn marshmallow nó hỗ trợ syntax:
```
sample_schema.jsonify(sample)  // số ít
samples_schema.jsonify(samples) // số nhiều
```